### PR TITLE
Add missing Tyk Pump configs to docs

### PR DESCRIPTION
**Pull request by n2linux**
### For internal users - please add a Jira DX PR ticket to the subject!
---
### Preview Link

[preview](https://deploy-preview-3038--tyk-docs.netlify.app/docs/nightly/tyk-configuration-reference/tyk-pump-configuration/csv/)

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

**Comment by letzya** [comment]
@n2linux is this PR close to ready? if yes, since there are new pages they need to be in another location but we, the DX team, will do it for you. Please let us know

**Comment by letzya** [comment]
@n2linux Hi, looks like an important PR, could you maybe get it reviewed so we can release it?

File: tyk-docs/content/tyk-configuration-reference/tyk-pump-configuration/elasticsearch.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+- `bulk_config`: Batch writing trigger configuration. Each option is an OR with each other:
+  - `workers`: Number of workers. Defaults to `1`.
+  - `flush_interval`: Specifies the time in seconds to flush the data and send it to ES. Default is disabled.
+  - `bulk_actions`: Specifies the number of requests needed to flush the data and send it to ES. Defaults to 1000 requests. If it is needed, can be disabled with `-1`.
+  - `bulk_size`: Specifies the size (in bytes) needed to flush the data and send it to ES. Defaults to 5MB. Can be disabled with `-1`.
+
+
+
+## Environment variables 
+```
```
Comment (letzya) [inline tyk-docs/content/tyk-configuration-reference/tyk-pump-configuration/elasticsearch.md]:
```suggestion
```bash
```

**Comment by dcs3spp** [comment]
@n2linux Could you get this peer reviewed and then we can merge and release

**Comment by n2linux** [comment]
🧉 @tbuchaillot and/or @matiasinsaurralde Could y'all review this and make sure these pump docs make sense so they can get merged? 🙏

File: tyk-docs/content/tyk-configuration-reference/tyk-pump-configuration/elasticsearch.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        "index_name": "tyk_analytics",
+        "elasticsearch_url": "http://localhost:9200",
+        "enable_sniffing": false,
+        "document_type": "tyk_analytics",
+        "rolling_index": false,
+        "extended_stats": false,
+        "version": "6"
+      }
+    },
+```
```
Comment (mativm02) [inline tyk-docs/content/tyk-configuration-reference/tyk-pump-configuration/elasticsearch.md]: Close this JSON

**Review by mativm02 — CHANGES_REQUESTED** [review]
I suggested a small change, anyways it looks good 👍

**Comment by dcs3spp** [comment]
Hi @n2linux, I have moved content files in the PR to tyk-docs/content/product-stack/tyk-pump/advanced-configurations/configure-data-sinks/ for new website. Will double check tomorrow, but I think we are close to merging and releasing this 👍

**Pull request merged**